src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class Ui {
+
+    public void start() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    public void end() {
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Bye. Hope to see you again soon!");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    public void addTaskMessage() {
+        System.out.println("     Got it. I've added this task:");
+    }
+
+    public void  printNumberOfTasks(int count) {
+        System.out.println("     Now you have " + count + " task" + ((count > 1) ? "s" : "") + " in the list.");
+
+    }
+
+    public void printCurrentTask(Task currTask) {
+         System.out.println("       " + currTask.toString());
```

Comment:
Should this line be indented by 1 space less? I noticed the same issue in several other places too.